client/src/modules/patients/record/bh-patient-financial-activity.html

```diff
@@ -79,6 +79,33 @@ <h5 ng-show="!$ctrl.status.isInGoodStanding" class="text-danger">
             (<span am-time-ago="$ctrl.data.aggregates.until"></span> <i class="fa fa-clock-o"></i>)
           </dd>
         </dl>
+
+        <hr/>
+        <h5 class="text-success" style="margin-top:0;">
```

Comment:
Can you remove the `text-success` here?  I'm not sure it makes sense to use here.

![image](https://user-images.githubusercontent.com/896472/78900392-b0d59500-7a6e-11ea-9506-0b5526069656.png)